.changeset/sharp-numbers-fix.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/cms-api": minor
+---
+
+Extend configuration options for `createAuthProxyJwtStrategy()`
+
+It's now possible to override the strategy and strategy options.
```

Comment:
@fraxachun Feel free to extend this 

https://github.com/vivid-planet/comet/pull/1159
https://github.com/vivid-planet/comet/pull/1160